docs/docs/api/authentication.md

```diff
@@ -75,6 +76,76 @@ curl -X POST \
 - In Success Response you will receive `token` and `refresh_token`.
 - Once you get the `token` using Login API, you can use that `token` for access other secure APIs.
 
+
+**Validate Access Token**
+---
+
+**CURL Sample**
+```
+curl -X POST \
+  https://portal.prancer.io/customer1/api/validate/accesstoken/ -H 'content-type: application/json' \
+  -d '{
+	"token" : "<Valid User Access Token>",
+	"customer_id" : "<Valid CustomerId>"
+}'
+```
+
+- **URL:** https://portal.prancer.io/customer1/api/validate/accesstoken/
+- **Method:** POST
+- **Header:**
+    - content-type: application/json
+- **Param:**
+```
+    {
+        "token" : "<Valid User Access Token>",
+        "customer_id" : "<Valid CustomerId>"
+    }
+```
+
+- **Explanation:**
+
+    `Required Fields`
+
+    - **token:** Valid User Access Token, generated from website.
+    - **customer_id:** Valid customer Id of user.
```

Comment:
@vatsalgit5118 put a description for customer_id:
customer_id is in the format of prancer-<your company name>. for example, if your company name is ABC, then the customer id would be prancer-abc